src/main/java/org/folio/consortia/utils/TenantContextUtils.java

```diff
@@ -1,42 +1,37 @@
 package org.folio.consortia.utils;
 
-import java.nio.charset.StandardCharsets;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
+import lombok.experimental.UtilityClass;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.collections4.MapUtils;
 import org.folio.spring.DefaultFolioExecutionContext;
 import org.folio.spring.FolioExecutionContext;
 import org.folio.spring.FolioModuleMetadata;
 import org.folio.spring.integration.XOkapiHeaders;
 import org.folio.spring.scope.FolioExecutionContextSetter;
 import org.springframework.messaging.MessageHeaders;
 
-import lombok.experimental.UtilityClass;
+import java.nio.charset.StandardCharsets;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
 @UtilityClass
+@Log4j2
 public class TenantContextUtils {
-  public static FolioExecutionContext getFolioExecutionContextCopyForTenant(FolioExecutionContext context,
-    String tenant) {
+  public static FolioExecutionContext getFolioExecutionContextCopyForTenant(FolioExecutionContext context, String tenant) {
     var headers = context.getAllHeaders() != null
       ? context.getAllHeaders()
       : new HashMap<String, Collection<String>>();
     headers.put(XOkapiHeaders.TENANT, Collections.singletonList(tenant));
 
     return new DefaultFolioExecutionContext(context.getFolioModuleMetadata(), headers);
   }
-  public static FolioExecutionContext getFolioExecutionContextCreatePrimaryAffiliationEvent(
-                                                                                  MessageHeaders headers,
-                                                                                  FolioModuleMetadata moduleMetadata) {
-    return getContextFromKafkaHeaders(headers, moduleMetadata);
-  }
 
-  public static FolioExecutionContext getFolioExecutionContextDeletePrimaryAffiliationEvent(
-                                                                                  MessageHeaders headers,
-                                                                                  FolioModuleMetadata moduleMetadata) {
-    return getContextFromKafkaHeaders(headers, moduleMetadata);
+  public static FolioExecutionContext createFolioExecutionContext(String centralTenantId, MessageHeaders headers,
```

Comment:
better to have the same parameters order between createFolioExecutionContext and getContextFromKafkaHeaders methods